python-checks/src/test/resources/checks/changeMethodContract/changeMethodContractParent.py

```diff
@@ -1,6 +1,6 @@
 from typing import Any
 
-class ParentClass(object):
+class ParentClass[T](object):
```

Comment:
Note: the issues were initially not raised due to `ParentClass` not being a proper generic class.